plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoTypeMapping.java

```diff
@@ -311,6 +311,15 @@ public void testArrayNulls()
         }
     }
 
+    @Test
+    public void testJSON()
```

Comment:
Rename to `testJson`.

Reply:
Yeah, but what are the considerations behind this? I thought JSON is like just any other acronyms (NASA? ASCII?)